Registry/prefab.tools.setreg

```diff
@@ -7,17 +7,21 @@
             {
                 "Processing":
                 {
-                    "Stack":
-                    {
-                        "PlayInEditor":
+                    "Stack": {
+                        "PlayInEditor": 
                         [
                             { "$type": "AzToolsFramework::Prefab::PrefabConversionUtils::EditorInfoRemover" },
                             { "$type": "AzToolsFramework::Prefab::PrefabConversionUtils::PrefabCatchmentProcessor" }
                         ],
-                        "GameObjectCreation":
+                        "TempSpawnables": 
```

Comment:
Perhaps call this "IntegrationTests" to be a bit more descriptive?
Note that this can also be moved to a `prefab.test.setreg` to avoid it getting mixed up with the regular stacks. (This because in the future it may be possible to select the stack in the editor and the test stack shouldn't show up in that case.)